controllers/Users/userController.js

```diff
@@ -0,0 +1,39 @@
+const User = require('../../lib/modelManagers/usermodel');
+
+const pagination = require('../../lib/utils/pagination/paginationHelper');
+
+/**
+ *
+ *
+ * @class Products
+ */
+class UsersController {
+  /**
+   * @static
+   * @returns { object } object
+   * @param {*} req
+   * @param {*} res
+   */
+  static async listUsers(req, res) {
+    const { limit, offset } = pagination(req.query);
+    try {
+      const searchResult = User.listAllUsers(limit, offset);
```

Comment:
consider changing `searchResult` to `users`

Reply:
done